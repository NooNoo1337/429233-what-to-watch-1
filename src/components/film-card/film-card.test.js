import React from 'react';
import renderer from 'react-test-renderer';

import FilmCard from './film-card.jsx';

const mockFilm =
  {
    id: 1,
    title: `Fantastic Beasts`,
  };

describe(`FilmCardComponent`, () => {
  it(`should render FilmCardComponent correctly`, () => {
    const tree = renderer
      .create(<FilmCard
        film={mockFilm}
        key={mockFilm.id}
        handleClick={() => {}}
        handleMouseEnter={() => {
        }}
      />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
