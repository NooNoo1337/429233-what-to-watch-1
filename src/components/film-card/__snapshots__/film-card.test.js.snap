// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilmCardComponent should render component correctly 1`] = `
<article
  className="small-movie-card catalog__movies-card"
  data-film-id={1}
>
  <video
    className="small-movie-card__image"
    height="175"
    muted={true}
    poster="https://es31-server.appspot.com/wtw/static/film/preview/what-we-do-in-the-shadows.jpg"
    width="280"
  >
    <source
      src="https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4"
      type="video/mp4"
    />
  </video>
  <h3
    className="small-movie-card__title"
  >
    <a
      className="small-movie-card__link"
    >
      Fantastic Beasts
    </a>
  </h3>
</article>
`;
